nodejs/package.json

```diff
@@ -11,6 +11,8 @@
   "dependencies": {
     "body-parser": "1.15.2",
     "express": "4.14.0",
-    "smooch-core": "2.1.1"
+    "smooch-core": "2.1.1",
+    "@babel/cli": "^7.0.0",
+    "superagent": "3.7.0"
```

Comment:
Why do we now need those libraries?

Reply:
removed. they were needed when the package was called from a "local" way. but htey would become peer dependencies otherwise.